### DISCUSSION TITLE
How to edit a directory in nerd-tree immediately?

### DISCUSSION THREAD

**Discussion by otakutyrant** [nvim-tree/nvim-tree.lua#2291] [Q&A] [answered]
Supposing `a` is a directory, `vim a` with default config will open the `a` in netrw immediatly. Now I want to use nerd-tree instead. But neovim still edit the directory in netrw after I have set `hijack_netrw = true` in the [minimal config](https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#clean-room-replication). Is there a bug or whether did I mistake what `hijack_netrw` means?

![2023-06-24-153240_1923x1080_scrot](https://github.com/nvim-tree/nvim-tree.lua/assets/64188229/fb0b8c78-614f-4285-9034-71ca0e661108)

**Comment by alex-courtis** [comment, marked as answer]
It sounds like you've encountered a bug. Please raise a [Bug Report](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) so that we may address this.

The recommended approach is to disable netrw completely `:help nvim-tree.disable_netrw`, that is done on line 1 and 2 of the minimal config.

`disable_netrw` and `hijack_netrw` have no effect when netrw is disabled. When netrw is enabled, `hijack_netrw` will replace netrw buffers with nvim-tree.

**Comment by alex-courtis** [comment]
You might like to look at [Wiki: Open At Startup](https://github.com/nvim-tree/nvim-tree.lua/wiki/Open-At-Startup) for some nice recipes.

**Comment by alex-courtis** [comment]
Continued in #2296
